### ISSUE TITLE
Unchecked runtime.lastError: Could not establish connection. Receiving end does not exist.

### ISSUE THREAD

**Issue by Alabate** [gaoliang/env-indicator#2]
Hi,

This extension is a really good idea and have a nice UI, thank for the work.

However, I've encountered a bug. For every website that I go to, I've the following error in the web console:

```
Unchecked runtime.lastError: Could not establish connection. Receiving end does not exist.
```

Most of the time, it seems to be harmless. But sometimes, it have an impact on the JS that seems to stop without any other errors. When it happen, I have to disable the extension in chrome (disabling just the indicator is not enough).

Chrome version: 88.0.4324.150
Extension version: 1.0.2

**Comment by gaoliang** [comment]
Thanks for your feedback, I will try to fix it after the Chinese New Year holiday.

**Comment by gaoliang** [comment]
I've released version [1.0.3](https://github.com/gaoliang/env-indicator/releases/tag/1.0.3) (still in review in the chrome shop), please test to see if it's fixed

**Comment by Alabate** [comment]
Yes, I confirm it's fixed. Thank you!
